lib/uklock/rwlock.c

```diff
@@ -1,3 +1,4 @@
+#include <uk/thread.h>
```

Comment:
Should be included in the "rwlock" header file, as "struct uk_thread" is used there as a struct field type.